kuma/static/styles/components/wiki/customcss.scss

```diff
@@ -578,5 +578,31 @@ pre span.comment {
     display: inline;
 }
 
+/*
+  Zone-related classes kept after the removal of zones.
+  (See https://bugzilla.mozilla.org/show_bug.cgi?id=1469029)
+ */
+
+/*
+  reference-values:
+
+  This gives an extra indent to a dl, when it's embedded inside another dl,
+  which comes up sometimes in the reference pages. The extra indent indicates
+  that the embedded dl is part of an embedded list, not the main list.
+ */
+dd > dl.reference-values {
+    padding-left: 40px;
+}
+
+/*
+  webextension-api-sidebar:
+
+  This applies a background color to the piece of a sidebar containing the
+  api items relevant to the current page. It helps people orient themselves
+  within the sidebar.
+ */
+li.webextension-api-sidebar {
```

Comment:
This could be `light-grey-background`, and be implemented as:

```
.light-grey-background {
	background-color: $grey-light;
}
```
Ref: https://github.com/mozilla/kuma/blob/master/kuma/static/styles/includes/_vars.scss#L29

Reply:
This requires that I also simultaneously update the `kumascript/macros/WebExtAPISidebar.ejs` macro, which I think has the only use of the `webextension-api-sidebar` class.

Reply:
👍 thanks @schalkneethling!

Reply:
Yeah, for this one renaming the class is easy. For `reference-values`, not so much :).